### PR TITLE
added card and avatar component

### PR DESCRIPTION
**Pull request by vishalg8454**

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    width: 22rem;
+    /* border: 2px solid red; */
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+    padding: 1rem;
+}
+.card-image{
+
```
Comment (irajbhinde) [inline components/card.css]: You can discard the unused css here

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+.card-btn-icon:hover{
+    background-color: var(--color-light-gray)
+}
+.card-badge{
+    position: absolute;
+    color: var(--color-white);
+    background-color: var(--color-primary);
+    padding: 4px;
+    border-radius: 0px 0px 4px 0px;
```
Comment (irajbhinde) [inline components/card.css]: You can use border-bottom-right-radius property here

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    opacity: 0.9;
+    align-self: center;
+    font-size: 2rem;
+}
+.card-horizontal{
+    flex-direction: row;
+    width: fit-content;
+}
+.card-container .card-container{
+    box-shadow: none;
```
Comment (irajbhinde) [inline components/card.css:77]: What's the use of this? Your card already has a border so why are you setting it to none.

File: docs.html [hunk 4]
```diff
@@ -128,12 +130,119 @@ <h2 class="component-title">Button</h2>
...
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <p class="component-description">Four sizes of text Avatar are available in sizes 'xsm', 'sm', 'md' and 'lg'.
+                </p>
+                <div class="avatar-output-container">
+                    <div class="avatar avatar-xsm avatar-text avatar-text-xsm">VG</div>
+                    <div class="avatar avatar-sm avatar-text avatar-text-sm">VG</div>
+                    <div class="avatar avatar-md avatar-text avatar-text-md">VG</div>
```
Comment (irajbhinde) [inline docs.html:236]: since you are using xsm sm md lg, then maybe you can discard the regular avatar-text class being used here.

**Review by irajbhinde — COMMENTED** [review]
Looks good, just look into the line comments for minor fixes.

File: components/card.css (hunk 3 above)
Comment (vishalg8454) [reply components/card.css:77]: This is used to remove shadow effect in the container inside the card.

File: docs.html (hunk 4 above)
Comment (vishalg8454) [reply docs.html:236]: I looked into it, the base class is needed.

**Review by Shailseen — APPROVED** [review]
Look all good for me.you can proceed to merge.

**Pull request merged**